src/components/Chat/ChatHeader/ChatHeader.ts

```diff
@@ -1,11 +1,12 @@
+import { compile } from 'pug'
 import Block from '../../../utils/Block'
-import template from './template.pug'
+import {chatHeaderTemplate} from './ChatHeader.tmpl'
 import Link from '../../../components/Link'
 import ChatController from '../../../controllers/ChatController'
 
 
 class ChatHeader extends Block {
-  constructor(props) {
+  constructor(props:any | undefined) {
```

Comment:
```suggestion
  constructor(props? :any) {
```
А вообще пропсы стоит типизировать конкретными интерфейсами для каждого компонента, чтобы ТС давал подсказки и с меньшей вероятностью было бы допустить ошибки